Source/EmbeddedBoundary/WarpXInitEB.cpp

```diff
@@ -82,8 +82,8 @@ WarpX::InitEB ()
 #ifdef AMREX_USE_EB
     BL_PROFILE("InitEB");
 
-#ifdef WARPX_DIM_RZ
-    amrex::Abort("Embedded Boundaries not implemented in RZ geometry");
+#if !(defined(WARPX_DIM_3D) || defined(WARPX_DIM_2D))
```

Comment:
Since RZ is not yet implemented, this should read:
```suggestion
#if !(defined(WARPX_DIM_3D) || defined(WARPX_DIM_XZ))
```
https://warpx.readthedocs.io/en/latest/developers/dimensionality.html